.github/workflows/publish-packages.yml

```
@@ -8,10 +8,18 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: 'read'
+      id-token: 'write'
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+      - name: Auth to GCP
+        id: 'auth'
+        uses: google-github-actions/auth@v0
```

```suggestion
        uses: google-github-actions/auth@v1
```

You can also use v1 here.

---

Apparently since then they've published a [v2](https://github.com/google-github-actions/auth), but I'm not sure it's backwards compatible.

---

This was copied from `tag-publish.yaml`. Looks like v2 is the latest version. Updated in https://github.com/DataBiosphere/terra-ui/pull/4709/commits/c48b6001c498d3068e00333c18ef6ac3a787003b.